tsconfig.json

```diff
@@ -23,7 +23,7 @@
 
     /* Additional Checks */
     "noUnusedLocals": false /* Report errors on unused locals. */,
-    "noUnusedParameters": true /* Report errors on unused parameters. */,
+    "noUnusedParameters": false /* Report errors on unused parameters. */,
```

Comment:
would prefer keeping this for easy visibility, since we have a couple of unused props in components as their functionalities aren't implemented yet

Reply:
Re-enabled in c8c9e11f7ec67ceb5e79785977ce53690b23b6f4. It spilled over cos I was trying to build the repo (enabling this will block builds).